geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/StartLocatorCommand.java

```diff
@@ -183,6 +188,8 @@ ResultModel doStartLocator(
 
     Properties gemfireProperties = new Properties();
 
+    StartMemberUtils.setPropertyIfNotNull(gemfireProperties, ConfigurationProperties.BIND_ADDRESS,
+        bindAddress);
```

Comment:
I thought it was odd that we had to call `setPropertyIfNotNull()` in this method (`doStartLocator()`) but we did not over in `StartServerCommand.doStartServer()`. Looks like it was already being called over in that other method.

Does that mean that before this code change here, we had a bug where gfsh `start locator` would ignore the bind address (before this PR)?